frontend/view-edit-lead/edit-lead-inputs/lead-contact-information-inputs.component.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import PhoneInput from "../../util/phone-input.component";
+
+export default function LeadContactInformationInputs(
+  props: LeadContactInformationInputsProps
+) {
+  const [phone, setPhone] = React.useState(props.lead.phone || "");
+  const [smsConsent, setSmsConsent] = React.useState(
+    props.lead.smsConsent || false
+  );
+  const [zip, setZip] = React.useState(props.lead.zip || "");
+
+  return (
+    <form onSubmit={handleSubmit} autoComplete="off" className="edit-form">
+      <div>
+        <label>
+          <span>Phone number</span>
+          <PhoneInput phone={phone} setPhone={setPhone} />
+        </label>
+      </div>
+      <div>
+        <label>
+          <span>Wants text messages</span>
+          <input
+            type="checkbox"
+            name="smsConsent"
+            checked={smsConsent}
+            onChange={evt => setSmsConsent(Boolean(evt.target.checked))}
+            className="checkbox"
+          />
+        </label>
+      </div>
+      <div>
+        <label>
+          <span>Zip Code</span>
+          <input
+            type="text"
+            value={zip}
+            onChange={evt => setZip(evt.target.value)}
+            autoComplete="new-password"
+            autoFocus
```

Comment:
We are we autofocusing this input? Usually you auto focus the first input in a form, which would be the sms consent in this case.